pkg/kubelet/server/server.go

```diff
@@ -41,6 +41,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
+	consts "k8s.io/apimachinery/pkg/util/remotecommand"
```

Comment:
Don't call this "consts", it's very confusing. (Maybe make a remotecommand/consts package if you really want to, but I don't think that's necessary.)

Multiple places.